main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/DotNetCoreVersion.cs

```diff
@@ -34,12 +34,14 @@ namespace MonoDevelop.DotNetCore
 {
 	class DotNetCoreVersion : IEquatable<DotNetCoreVersion>, IComparable, IComparable<DotNetCoreVersion>
```

Comment:
wouldn't it be more interesting to keep track of which net standard versions are supported by the .net sdks? this way we don't need to compare the sdk's version number, because not every .net sdk supports every net standard version.

Reply:
looking back, I really don't know why I changed this version from (2, 1, 602) to (2, 1, 30)!. The matrix of supported versions etc  has never been a strong point for me.   I was just going for the minimum changes to the existing code (and way of doing things) that fixed the netstandard issue.  @gluckez by all means update if you have better understanding

Reply:
@hwthomas If it helps, I will certainly not stop you from merging it ;) And I'll have a look tonight to see if the knowledge about the supported frameworks and netstandard versions can be centralized.

Reply:
@gluckez Many thanks - I appreciate that.  
So far, I'm afraid I've been focussing on getting `net5ff` and `netstandard` projects working, and not being too concerned about improving the state of the codebase!

Reply:
@hwthomas I guess that's why I'm here xD I'm not too familiar with the codebase yet, but if I can make small improvements while learning, I'll give it a go.